woocommerce/woocommerce-android
# Fix the end sale date button removal

Fixes #2247.

This PR fixes the `Remove end date` button in the Product pricing screen.

To test:
1. Go to a product detail screen
2. Tap on the Price card
3. Switch on the Schedule sale switch
4. Select a start date & end date
5. Notice the `Remove end date` is visible
6. Tap on the button
7. Notice the end date is gone and the button is hidden
